### PR TITLE
Makes Syndicate code words more generic and natural sounding

### PR DESCRIPTION
**Pull request by tipoima**
## What Does This PR Do
Removes drinks and crew names from possible code words.
Replaces specific room names with a smaller list of generic areas ("Cargo lobby" -> "Cargo", "Medical Treatment Center" -> "Medbay")
Adds a few extra nouns and changes plural ones into singular (why was one of code words specifically "friendships" anyway?)
Removes the possibility of selecting only 2 code words and makes it more likely to select 4.

## Why It's Good For The Game
In my experience, code words often end up extremely obvious for both the crew and other traitors. 

Drinks: Nobody really talks about drinks outside of bar, and not a lot of people go to bar in the first place. 
Names: Many people either have nicknames/middle names as part of their name (Shawn 'Archibald' Morgan) have overly long names (Bodhi The Hermit Of Cold Caves), or are entirely unpronounceable gibberish (Volxuqzixzaoo*xuquumzixqr Quumquumxuqxu-uuzixzaoo!) People rarely say full names, instead only using first name or shortening it in other ways.
Locations: "Aft-Port Secondary Maintenance", words only said when copying a location from suit sensors monitor. 

As they are now, half the rounds start with a blatantly obvious and cringeworthy code word exchange, rather than any attempt to use them in a natural conversation.
Replacing them allows for actual subtle code word exchange, while also making traitors actually think about regular crew using code words accidentally.

A couple added generic nouns help dilute the more unusual ones like "integrity" and "brilliance", as they also get mildly metagamed. 

## Testing
Compiled, updated code words show up

## Changelog
:cl:
del: Removed code words for drinks and crew member names
tweak: Replaced code words for room names with more generic ones
/:cl:

### CODE REVIEW

File: code/__HELPERS/name_helpers.dm [hunk 1]
```diff
@@ -132,21 +132,16 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
...
-	var/drinks[] = list("vodka and tonic","gin fizz","bahama mama","manhattan","black Russian","whiskey soda","long island tea","margarita","Irish coffee"," manly dwarf","Irish cream","doctor's delight","Beepksy Smash","tequila sunrise","brave bull","gargle blaster","bloody mary","whiskey cola","white Russian","vodka martini","martini","Cuba libre","kahlua","vodka","wine","moonshine")
-	var/locations[] = length(SSmapping.teleportlocs) ? SSmapping.teleportlocs : drinks//if null, defaults to drinks instead.
+	var/nouns[] = list("love","hate","anger","peace","pride","sympathy","bravery","loyalty","honesty","integrity","compassion","charity","success","courage","deceit","skill","beauty","brilliance","pain","misery","belief","dream","justice","truth","faith","liberty","knowledge","thought","information","culture","trust","dedication","progress","education","hospitality","leisure","trouble","friendship","relaxation","laser","robot","box","crate","camera","wire","glass","plasma","gold","iron","titanium","diamond")
```
Comment (warriorstar-orion) [inline code/__HELPERS/name_helpers.dm:142]: could you pull these into text files in `config/names` (e.g. `config/names/codeword_nouns.txt` and `config/names/codeword_locations.txt`) and load them into a global list in `code/_globalvars/lists/names.dm`?

Comment (tipoima) [reply code/__HELPERS/name_helpers.dm]: I think I did it? Seems to work fine

File: config/names/codeword_locations.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+robotics
+science
+xenobiology
+genetics
+medbay
+chemistry
+morgue
+engineering
+atmospherics
+maintenance
```
Comment (warriorstar-orion) [inline config/names/codeword_locations.txt:22]:
```suggestion
armory
arrivals
atmospherics
bar
botany
bridge
brig
cargo
chemistry
departures
dormitories
engineering
genetics
holodeck
kitchen
library
maintenance
medbay
morgue
robotics
science
xenobiology
```

Have these sorted

File: config/names/codeword_nouns.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+box
+crate
+camera
+wire
+glass
+plasma
+gold
+iron
+titanium
+diamond
```
Comment (warriorstar-orion) [inline config/names/codeword_nouns.txt:51]:
```suggestion
anger
beauty
belief
box
bravery
brilliance
camera
charity
compassion
courage
crate
culture
deceit
dedication
diamond
dream
education
faith
friendship
glass
gold
hate
honesty
hospitality
information
integrity
iron
justice
knowledge
laser
leisure
liberty
love
loyalty
misery
pain
peace
plasma
pride
progress
relaxation
robot
skill
success
sympathy
thought
titanium
trouble
trust
truth
wire
```

and these

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]

**Review by warriorstar-orion — APPROVED** [review]
Code LGTM

**Comment by tipoima** [comment]
A round yesterday had code words "chop" and "chop-chop" (a player's name), in case anyone else isn't sold on this.

It also had "Special Operations Officer" which probably isn't intended but isn't something I know how to fix without hardcoding jobs into the config

**Review by lewcc — APPROVED** [review]
Code looks good to me, pending config update

**Review by AffectedArc07 — APPROVED** [review]
Turns out I made these config changes like months ago

**Pull request merged**